File: Arreglos/MatricesMultidimensionales/Array2D.java
// Implementación en Java de una matriz bidimensional
// Autor: Rivera Cota Jesús Crispín
// FACULTAD DE INGENIERÍA LOS MOCHIS
// Ing. de Software 201

public class Array2D {
    public static void main(String[] args) {
        // Declaración e inicialización de la matriz bidimensional
        int[][] TwoDimensionlArray = {
            {1, 2, 3},
            {4, 5, 6},
            {7, 8, 9}
        };

        System.out.println("Los elementos de la matriz son: ");
        // Recorrido por filas
        for (int[] row : TwoDimensionlArray) {
            // Recorrido por elementos dentro de la fila
            for (int element : row) {
                System.out.print(element + " "); // Mostrando los elementos de la fila separados por espacios
            }
            System.out.println(); // Ir a la siguiente línea después de mostrar una fila
        }
    }
}

File: Arreglos/MatricesMultidimensionales/Array2D.py
# Implementación en python de una matriz bidimensional
# Autor: Rivera Cota Jesús Crispín
# FACULTAD DE INGENIERÍA LOS MOCHIS
# Ing. de Software 201

TwoDimensionlArray = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]
print("Los elementos de la matriz son: ")
for row in TwoDimensionlArray:
    for element in row:
        print(element, end=" ")#mostrando los elementos de la fila separandos por espacios
    print() #ir a la siguiente linea después de mostrar una  fila